### ISSUE TITLE
Consider not excluding `.well-known/*` from publishing

### ISSUE THREAD

**Issue by tiesmaster** [dotnet/sdk#25924]

**Comment by baronfel** [comment]
I think this is completely reasonable, but would be something that's more directly applicable to the Web SDK rather than the base set of default includes. @DamianEdwards what do you think?

**Comment by DamianEdwards** [comment]
Yep agreed, makes sense to have the web SDK support this. @vijayrkn could we get this added?
